cityofaustin/vision-zero
# Crash details: show banner when crash is outside of AFP or on a private drive

## Associated issues

https://github.com/cityofaustin/atd-data-tech/issues/20007

I also replaced the nextjs favicon with the vze one but I can remove that if it was premature

## Testing

**URL to test:** 
local

**Steps to test:**

You will need to delete the crashesListViewQueryConfig entry from your localstorage if you have not already been testing Johns other PRs. 
Go to any crash page. If the crash is not on a private drive and in Austin full purpose, you will not see any location banner at the top. 
Toggle the crash to be on a private drive. The banner appears! 
Toggle the crash to not be on a private drive. Move the location using the map to be outside of Austin.  Banner appears again. 

![Screenshot 2024-12-05 at 8 17 21 AM](https://github.com/user-attachments/assets/27c811b0-55a7-470e-84c3-46d1124fdf2b)


---

#### Ship list
